asynq/tests/test_asynq_to_async.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 Quora, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import asyncio
+
+import asynq
+
+
+async def f3():
```

Comment:
Can you add some tests involving asynq contexts (with `__pause__` and `__resume__`)? I'm afraid those currently won't work correctly.